test/unit/cxx_test_shmem_atomic_inc.cpp

```diff
@@ -90,6 +119,7 @@ enum op { INC = 0, ATOMIC_INC, CTX_ATOMIC_INC, FINC, ATOMIC_FETCH_INC,
             rc = EXIT_FAILURE;                                          \
           }                                                             \
           break;                                                        \
+        SHMEMX_NBI_OPS_CASES(OP, TYPE)                                  \
```

Comment:
Adding the switch cases like this is great to read.

Reply:
Yeah, you can't have preprocessor conditions (check for ENABLE_SHMEMX_TESTS) in a macro definition, so you end up needing another macro to capture the conditional.  Otherwise I would have preferred put the code in here directly.